File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -1834,3 +1834,153 @@ document.Close(True)
 {% endhighlight %}
 
 {% endtabs %}
+
+## Inject custom CSS
+
+The Blink rendering engine supports the injection of custom CSS to be applied to HTML or a URL before rendering it into a PDF document using the 'Css' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class 
```

Comment:
The Blink rendering engine supports the injection of custom CSS to be applied to the HTML or a URL before rendering it into a PDF document using the 'CSS' property of the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class.